charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,80 @@
+
+# Project Name
+HCA Data Processing Service
+
+## Description
+
+The data processing service is a centralized space to perform all transformative functionality to scientific data ingested into the Human Cell Atlas Data Coordination Platform (HCA DCP).
+
+## Definitions
+
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
```

Comment:
* `project, but`
* nit: Many of the items in lists here are not full sentences. I'd remove the period from those that aren't. Apply this rule consistently throughout the doc.

Reply:
thanks!